Eevee.Sleep.Bot/Utils/ExcludedRoleNames.cs

```diff
@@ -0,0 +1,8 @@
+namespace Eevee.Sleep.Bot.Utils;
+
+public static class ExcludedRoleNames
+{
+    public static readonly string[] RoleNames = [
+        "@everyone",
```

Comment:
Does Discord handle `@everyone` like a role? Don't think this would ever get used.

And if we ever need to track role exclusions, use role IDs instead if possible

Reply:
`@everyone` isn't really a role, you can just remove this file I think, it won't get tracked anyway

Reply:
When adding a role for display, once the other roles have to be deleted, they need to be excluded!

Reply:
Do you have an example? Not sure if I get this

Reply:
Implemented by deleting only the roles being tracked
[0efdb13](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/0efdb13c967cbcb919883869038792ccfef7bf87)